README.md

```diff
@@ -198,7 +199,105 @@ The library adds a `form-submitted` to the submitted form. You can use it to sty
   }
 }
 ```
+- `controlErrorComponent` - Optional. Allows changing the default component that is used to reflect 
+  the errors. This component should implement the `ControlErrorComponent` interface.  Defaults to
+  `DefaultControlErrorComponent`.
+
+  For example:
+  ```ts
+  // Custom error component that will replace the standard DefaultControlErrorComponent.
+  // Use an HTML fragment that suits your UI framework of choice. Example below shows
+  // error text wrapped in an <ion-item> which best serves a typical Ionic form layout
+  // where each form field is enclosed in an <ion-item>, all of which are wrapped in
+  // an <ion-list>.
+  @Component({
+    template: `
+    <ion-item lines='none' class='ion-text-wrap'>
```

Comment:
use double quotes for each attribute value, please.

Reply:
Noted. Is that Angular convention or personal preference?

Reply:
HTML convention.